docs/reference/setup/add-nodes.asciidoc

```diff
@@ -14,8 +14,8 @@ functional but is at risk of data loss in the event of a failure.
 image::setup/images/elas_0202.png["A cluster with one node and three primary shards"]
 
 You add nodes to a cluster to increase its capacity and reliability. By default,
-a node is both a data node and eligible to be elected as the master node that
-controls the cluster. You can also configure a new node for a specific purpose,
+a node is both a data node and eligible to act as the manager node that
```

Comment:
This is a bit of a weird sentence, nodes have many roles by default. At least I'd say to remove the eligibility-to-become-manager bit and instead talk about the control role.